Added the following recommendations to Azure Site Recovery. (#263)
Azure/Azure-Proactive-Resiliency-Library: docs/content/services/compute/site-recovery/_index.md

Reviewer: As these are new recommendations, you will also need to update the table toward the top of this page.

{{< /collapse >}}

<br><br>

> ### ASR-3 - Monitor Source VMs for high data change rates

Reviewer: ```suggestion
### ASR-3 - Monitor source VMs for high data change rates
```

**Category: Disaster Recovery**

**Impact: High**

**Guidance**

> Azure Site Recovery has limits on data change rates, depending on the type of disk. To see if this problem is recurring or temporary, find the data change rate of the affected virtual machine. Go to the source virtual machine and monitor the **OS Disk Write Bytes/Sec** and **Data Disk Write Bytes/Sec** metrics for OS and Data disks.

Reviewer: ```suggestion
Azure Site Recovery has limits on data change rates, depending on the type of disk. To determine if this is a recurring or temporary problem, find the data change rate of the affected virtual machine. Go to the source virtual machine and monitor the **OS Disk Write Bytes/Sec** and **Data Disk Write Bytes/Sec** metrics for OS and Data disks.
```

- [Azure VM Disaster Recovery - High Churn Support](https://learn.microsoft.com/en-us/azure/site-recovery/concepts-azure-to-azure-high-churn-support)

**Resource Graph Query/Scripts**

{{< collapse title="Show/Hide Query/Script" >}}

{{< code lang="sql" file="code/asr-3/asr-3.kql" >}} {{< /code >}}

{{< /collapse >}}

<br><br>

### ASR-4 - Ensure sufficient resource quotas in the target region

**Category: Disaster Recovery**

**Impact: High**

**Guidance**

Make sure your subscription is enabled to create Azure VMs in the target region that you plan to use as your disaster recovery (DR) region. Your subscription needs sufficient quota to create VMs of the necessary sizes. By default, Site Recovery chooses a target VM size that's the same as the source VM size.

If the target location has a capacity constraint, disable replication to that location. Then, enable replication to a different location where your subscription has sufficient quota to create VMs of the required sizes.

**Resources**

- [Azure resource quota issues](https://learn.microsoft.com/en-us/azure/site-recovery/azure-to-azure-troubleshoot-errors#azure-resource-quota-issues-error-code-150097)

<br><br>

> 
> 

Reviewer: ```suggestion
```
